giant/plugins/client/java_android/java_android/java_android/templates/GiantTemplate.iml.jinja

```diff
@@ -1,22 +0,0 @@
-{%- import 'variables-android.jinja' as android with context -%}
```

Comment:
Please do not remove these files. This project is intended to not be a single purpose plugin for us, but should be generally consumable by the public.